modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/BaseSamlTestCase.java

```diff
@@ -618,6 +620,8 @@ private void _setupMetadata() throws Exception {
 
 		fileSystemKeyStoreManagerImpl = new FileSystemKeyStoreManagerImpl();
 
+		samlHttpRequestUtilImpl = new SamlHttpRequestUtilImpl();
+
```

Comment:
Hey @GabrielHenriquedev, move this variable to the `MetadataManagerImplTest.java` class instead. Thx